### ISSUE TITLE
Scenario system is impractical

### ISSUE THREAD

**Issue by mikron-ia** [mikron-ia/rpg-hub#213]
Scenario system should be made less clunky. It currently uses `Description` objects, which are fine for something that requires fine control over language and accessibility, but scenarios do not - they are simple notes for GMs and no one else.

**Comment by mikron-ia** [comment]
Since `Scenario` had existed for a while, just removing all the `Description` records is not an option. What _is_ and option is adding a simple text field (similar to the concept delineated in #71) for the GM to use and display it separately from the `Description` list; the user can use what they prefer.

If #412 is done first, this task should also add display of this note / those notes on the front; if not, add this part to the task.
